universe/ValueRef.cpp

```diff
@@ -97,11 +97,11 @@ namespace {
 
     // Generates a debug  trace that can be included in error logs, augmenting the ReconstructName() info with
     // additional info identifying the object references that were successfully followed.
-    std::string TraceReference(const std::vector<std::string>& property_name, ValueRef::ReferenceType ref_type,
+    std::string TraceReference(const std::vector<std::string>& property_names, ValueRef::ReferenceType ref_type,
```

Comment:
As far as I recall, it is just a single property name, not multiple property names. The vector contains the parts of the name broken up, eg. `Source.Planet.Population` -> `{Source, Planet, Population}`

Reply:
Changed to renaming shadowing var to `property_name_part`, consistent with `ReconstructNames`